### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by aaronfriedman6**

### CODE REVIEW

**Review by mwbenowitz — APPROVED** [review]
This looks solid. Only had a few comments about readability/clarity, but overall had no issue following the logic.

One more general question was if we wanted to "fuzz" some of the error states more. Right now if there isn't an exact match an error will get raised, is that what we want? (e.g. if it's off by one, how severe do we assess that error? If there are consistent issues it's worth checking out for sure. Maybe some kind of scale of messages is warranted at some point?)

File: alarm_controller.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                          build_redshift_circ_trans_query,
+                          build_redshift_deleted_patrons_query,
+                          build_redshift_new_patrons_query,
+                          build_redshift_pc_reserve_query,
+                          build_sierra_circ_trans_query,
+                          build_sierra_deleted_patrons_query,
+                          build_sierra_new_patrons_query)
+
+
+class AlarmController:
```
[the remark this answers is not in the document]
Comment (aaronfriedman6) [reply alarm_controller.py:19]: Yeah I think if we decide to add more tests in the future then we can refactor but it's fine for now.

**Comment by aaronfriedman6** [comment]
Replying to your comment above: I thought about some fuzz but theoretically the number of records should match exactly. If I find that a couple records are frequently being dropped (and this seems arbitrary/isn't a problem), then I'll add some leeway but I think being strict is a good starting point.

**Comment by mwbenowitz** [comment]
Great, starting out strict and then relaxing things if it turns out to be unnecessarily noisy totally makes sense to me!

**Pull request merged**